--- libdvdnav-0.1.9/configure.ac.orig	Tue May 20 23:10:58 2003
+++ libdvdnav-0.1.9/configure.ac	Wed Oct 15 11:45:20 2003
@@ -150,7 +150,7 @@
 dnl cflags
 dnl ---------------------------------------------
 dnl Common cflags for all platforms
-CFLAGS="$CFLAGS -O3 -Wall -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE"
+CFLAGS="$CFLAGS -Wall -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE"
 DEBUG_CFLAGS="$CFLAGS -g -DDEBUG"
 
 AC_SUBST(DEBUG_CFLAGS)
