diff -U 3 -H -d -r -N -- libdvdnav-4.1.3~/misc/dvdnav.pc.in libdvdnav-4.1.3/misc/dvdnav.pc.in
--- libdvdnav-4.1.3~/misc/dvdnav.pc.in	2008-07-13 23:13:40.000000000 +0200
+++ libdvdnav-4.1.3/misc/dvdnav.pc.in	2008-10-27 01:17:34.445212997 +0100
@@ -1,7 +1,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
-includedir=@includedir@
+includedir=@includedir@/dvdnav
 
 Name: libdvdnav
 Description: DVD Navigation library
